internal/ethapi/transaction_args.go

```diff
@@ -286,6 +286,10 @@ func (args *TransactionArgs) setLondonFeeDefaults(ctx context.Context, head *typ
 
 // setBlobTxSidecar adds the blob tx
 func (args *TransactionArgs) setBlobTxSidecar(ctx context.Context, b Backend) error {
+	isCancun := b.ChainConfig().IsCancun(b.CurrentHeader().Number, b.CurrentHeader().Time)
+	if !isCancun && (args.Blobs != nil || args.Commitments != nil || args.Proofs != nil || args.BlobHashes != nil) {
+		return errors.New("blobs are only valid after Cancun is active")
+	}
```

Comment:
I think this should be removed. But just so I understand, this would happen if the user passes in sidecar, but geth is not in cancun mode? 
IF so, I think we should let the user proceed.

Reply:
Removed. Exactly if user sends in blobs to compute the proofs and commitments, however chain is still behind cancun.